tests/test-definition.yaml

```diff
@@ -70,6 +77,10 @@ tests:
   - name: iceberg
     dimensions:
       - spark
+  - name: delta
```

Comment:
nit: I would prefer to call this test-case (also the folder) `delta-lake`, as I have the feeling this term is used more widely

Reply:
Done https://github.com/stackabletech/spark-k8s-operator/pull/355/commits/61a82ff73554ff8e7bef2499820196e7e33f39dd